content/en/docs/tasks/administer-cluster/migrating-from-dockershim/migrating-telemetry-and-security-agents.md

```diff
@@ -8,31 +8,20 @@ weight: 70
 
 <!-- overview -->
 
-With Kubernetes 1.20 dockershim was deprecated. From the
-[Dockershim Deprecation FAQ](/blog/2020/12/02/dockershim-faq/)
-you might already know that most apps do not have a direct dependency on runtime hosting
-containers. However, there are still a lot of telemetry and security agents
-that has a dependency on docker to collect containers metadata, logs and
-metrics. This document aggregates information on how to detect these
-dependencies and links on how to migrate these agents to use generic tools or
-alternative runtimes.
+Docker is being deprecated. You can find a deprecation FAQ at [Don't Panic: Kubernetes and Docker](https://kubernetes.io/blog/2020/12/02/dont-panic-kubernetes-and-docker/). Most apps do not have a direct dependency on runtime hosting containers. However, there are still a lot of telemetry and monitoring agents that has a dependency on docker to collect containers metadata, logs and metrics. This document aggregates information on how to detect these dependencies and links on how to migrate these agents to use generic tools or alternative runtimes.
 
 ## Telemetry and security agents
 
-There are a few ways agents may run on Kubernetes cluster. Agents may run on
-nodes directly or as DaemonSets.
+There are a few ways agents may be run on kubernetes cluster. Agents have a direct 
+dependency on Docker when they run on nodes or as daemonsets. Some telemetry and 
+security agents may rely on certain logs or metrics specific for Docker that would not 
+be available any longer. This document does not cover these cases.
```

Comment:
```suggestion
Within a Kubernetes cluster there are a few different ways to run telemetry or security agents.
Some agents have a direct  dependency on Docker Engine when they as DaemonSets or
directly on nodes.
```

I would move “For example, some telemetry and security agents rely on accessing logs or
metrics that are specific to Docker Engine” after the question “Why do some telemetry agents communicate with Docker Engine?”